test/framework/cluster_proxy.go

```diff
@@ -46,7 +46,7 @@ import (
 
 const (
 	retryableOperationInterval = 3 * time.Second
-	retryableOperationTimeout  = 1 * time.Minute
+	retryableOperationTimeout  = 3 * time.Minute // required higher value, otherwise failed at https://github.com/kubernetes-sigs/cluster-api/blob/63a959a1af1787063e5db0e279f17cf4e370b0ed/test/framework/machine_helpers.go#L112
```

Comment:
Do we know why? Can we add this to the comment?

(let's also move this comment on line above, the line is really really long)

Reply:
Let's try again with the old default and see 🤔  It failed in the linked code, but maye due to the changes for preload this is not relevant anymore 👍

Reply:
Sounds good!

Reply:
increased the timeout again because it got hit. I improved the comment, PTAL.

It may improve when we have etcd learner mode..